### DISCUSSION TITLE
Use Tin to quickly set up Grafana in minutes

### DISCUSSION THREAD

**Discussion by fangdajiang** [grafana/grafana#43426] [General]
Tin is a product of TeamCode that can generate executable applications instantly without manual environment configuration. Users can click to run project directly in the cloud. 

### Login to Tin and clone the Grafana app

1. Click the Tin URL of Grafana.

   https://www.teamcode.com/tin/clone?applicationId=270273627750297600

   Users who have not logged in should sign up first.

2. Log in to Tin with your email, GitHub account, or phone number.

![img](https://static01.teamcode.com/docs/202112211833751.png)

3. (Optional) After logging in, you can change the "Application Name" or "Tag".
4. Click "Clone" button to clone the Tin app to your workspace.

> **Note:** Tin will give new users a free trial for a certain length of time, which is determined by the instance size. After the free trial time, the Tin app will automatically stop working. You will not be charged for the whole process. 

![img](https://static01.teamcode.com/docs/202112211833030.png)

5. Click "RUN NOW" to initiate the application.

![img](https://static01.teamcode.com/docs/202112211833077.png)

6. Click the "Endpoint List" button to go to the Endpoint list, with the entry points for all apps displayed. Click the URL of entry point.

![img](https://static01.teamcode.com/docs/202112211833949.png)

![img](https://static01.teamcode.com/docs/202112211833961.png)

### Now you can log in to Grafana to use it.

![img](https://static01.teamcode.com/docs/202112211833110.png)

You can also use the Tin to quickly set up other excellent open source projects, see the guide: https://www.teamcode.com/docs/en-US/tin/create-tin

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
